.github/workflows/CI.yml

```diff
@@ -0,0 +1,148 @@
+name: CI
+on:
+  # always execute docker build when something is pushed to master or release-* branches
+  push:
+    branches:
+      - 'master'
+      - 'release-*'
+  # in addition, execute for pull requests to those branches
+  pull_request:
+    branches:
+      - 'master'
+      - 'release-*'
```

Comment:
If you're gonna use `standard-version` for releases I don't think you should trigger builds on `release-*` but rather on maintenance branches like `[0-9]+.[1-9][0-9]*.x`

Reply:
Thanks. For now we will keep building from `release-*` branches.